dags/vz_google_sheets.py

```diff
@@ -42,7 +42,7 @@ def task_fail_slack_alert(context):
 #to read the python script for pulling data from google sheet and putting it into tables in postgres
 try:
   sys.path.append('/etc/airflow/data_scripts/vision_zero/')
-  from schools import pull_from_sheet
+  from schools2020 import pull_from_sheet
```

Comment:
I think this is going to fail, since the schools file didn't get renamed

Reply:
good catch, thanks!